nemo/collections/tts/modules/ssl_tts.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
IIRC this should have "and affiliates"

Reply:
yeah, I saw mixed usage in our code base. But I will update relevant files in this PR.